src/apis/impl/HealthcareElementApiImpl.ts

```diff
@@ -214,4 +216,34 @@ export class HealthcareElementApiImpl implements HealthcareElementApi {
         return HealthcareElementMapper.toHealthcareElement(updatedHealthElement)!
       })
   }
+
+  async giveAccessToMany(healthElements: Array<HealthcareElement>, delegatedTo: string): Promise<Array<string>> {
```

Comment:
We need to think deeper about the giveAccessToMany, let's discuss about it together